### PR TITLE
RAC-464: Use illu as svg

### PR DESCRIPTION
**Pull request by ValentinMumble**
**Description (for Contributor and Core Developer)**

Now loading SVG DSM illustrations as images, reducing the DSM build time (48s -> 15s) and reducing the PIM vendor bundle size (11MB -> 7MB) :dabs:
Before:
image
image

After:
image
image

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by StevenVAIDIE — CHANGES_REQUESTED** [review]
NOT GTM you should do all the illustration

**Review by juliensnz — APPROVED** [review]

File: akeneo-design-system/static/illustrations/Export.svg [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+                        <path d="M24.7959,32.1223 C24.7959,32.1223 30.7099,28.8093 29.5089,23.3763 C30.3079,24.0183 32.4459,25.9773 33.2259,26.9843 C33.2389,28.0513 33.4879,32.2843 27.6759,35.1933 C26.6339,34.1403 25.7509,32.8203 24.7959,32.1223" id="Fill-95" fill="#FFFFFF"></path>
+                        <path d="M25.6338,32.176 C26.0598,32.579 26.4688,33.063 26.8948,33.566 C27.1808,33.905 27.4748,34.252 27.7788,34.578 C32.7188,31.958 32.7488,28.431 32.7288,27.163 C32.1668,26.481 31.0578,25.408 30.1648,24.604 C30.2838,28.53 27.0088,31.222 25.6338,32.176 L25.6338,32.176 Z M27.5748,35.803 L27.3208,35.545 C26.8998,35.119 26.5088,34.657 26.1308,34.211 C25.5678,33.545 25.0338,32.916 24.5008,32.526 L23.8738,32.068 L24.5508,31.687 C24.6078,31.655 30.1258,28.485 29.0208,23.485 L28.7138,22.096 L29.8218,22.987 C30.6758,23.672 32.8208,25.645 33.6208,26.679 L33.7238,26.811 L33.7258,26.978 C33.7928,30.791 31.8318,33.672 27.8998,35.641 L27.5748,35.803 Z" id="Fill-97" fill="#52267D"></path>
```
Comment (StevenVAIDIE) [inline akeneo-design-system/static/illustrations/Export.svg:68]:
```suggestion
                        <path d="M25.6338,32.176 C26.0598,32.579 26.4688,33.063 26.8948,33.566 C27.1808,33.905 27.4748,34.252 27.7788,34.578 C32.7188,31.958 32.7488,28.431 32.7288,27.163 C32.1668,26.481 31.0578,25.408 30.1648,24.604 C30.2838,28.53 27.0088,31.222 25.6338,32.176 L25.6338,32.176 Z M27.5748,35.803 L27.3208,35.545 C26.8998,35.119 26.5088,34.657 26.1308,34.211 C25.5678,33.545 25.0338,32.916 24.5008,32.526 L23.8738,32.068 L24.5508,31.687 C24.6078,31.655 30.1258,28.485 29.0208,23.485 L28.7138,22.096 L29.8218,22.987 C30.6758,23.956 32.8208,25.645 33.6208,26.679 L33.7238,26.811 L33.7258,26.978 C33.7928,30.791 31.8318,33.672 27.8998,35.641 L27.5748,35.803 Z" id="Fill-97" fill="#52267D"></path>
```

**Review by StevenVAIDIE — APPROVED** [review]
My computer does not freeze anymore when i load this page, please do not merge this PR :pray:

File: akeneo-design-system/static/illustrations/Export.svg (hunk 1 above)
Comment (ValentinMumble) [reply akeneo-design-system/static/illustrations/Export.svg:68]: Ah yes thank you!!!! Of course!

**Pull request merged**